### DISCUSSION TITLE
Star Wars Titling

### DISCUSSION THREAD

**Discussion by ghost** [jellyfin/jellyfin#10611] [Q&A] [answered]
Hello, I want to title all the star wars films from episode 1 through 6 including rogue one. Can you give some advice how to title them? I found the wiki page a bit confusing. So please help me out. Thank you, it is my first time.

This is how i currently titled the movies, is this correct or not?
![starwarsss](https://github.com/jellyfin/jellyfin/assets/108656189/0ff6e8fb-35a1-42fc-b66d-20218a26b8f1)

**Comment by TheMelmacian** [comment]
At first glance it looks OK to me.
I always check the title and year on https://www.themoviedb.org. There you can also find alternative titles: https://www.themoviedb.org/movie/11-star-wars/titles.
All Star Wars movies: https://www.themoviedb.org/collection/10-star-wars-collection

I recommend to put each Movie in a subfolder, especially if you save .nfo files and pictures in the folder alongside the movie. This is my personal preference but it will also be easier to add additional versions of a movie (4k, Specialized, Despecialized etc.) in the future, without the need to manually group them: https://jellyfin.org/docs/general/server/media/movies#multiple-versions-of-a-movie

**Reply by ghost** [reply]
so something like this?
![the fix](https://github.com/jellyfin/jellyfin/assets/108656189/d1babd81-c825-41a4-9f71-cad52e2c11bd)

**Reply by TheMelmacian** [reply, marked as answer]
Yes, looks good.

**Reply by ghost** [reply]
thank you
